ui/app/components/app/modals/customize-nonce/index.scss

```diff
@@ -0,0 +1,61 @@
+.customize-nonce-modal {
+  padding-left: 24px;
+  padding-right: 18px;
+  display: flex;
+  flex-flow: column nowrap;
+
+  &__main-header {
+    display: flex;
+    flex-flow: row;
+    justify-content: left;
+    align-items: left;
+    position: relative;
+    padding-top: 24px;
+  }
+
+  &__close {
+    @include H4;
+
+    color: #24292e;
+    background: none;
+    position: absolute;
+    right: 10px;
+    width: 0.75rem;
+    height: 0.75rem;
+    cursor: pointer;
+  }
+
+  &__description {
+    padding-top: 8px;
+  }
+
+  &__content {
+    padding-top: 20px;
+  }
+
+  &__header {
```

Comment:
This can be achieved by adding ```display={DISPLAY.FLEX}``` to the Box component. The default flex-flow for flexbox is `row` so its not needed. This class can then be removed entirely.

https://github.com/MetaMask/metamask-extension/blob/b68233d6189d36b046c4627d547d3b8e328bc5cd/ui/app/helpers/constants/design-system.js#L90-L98